### PR TITLE
feat(suite): autostart

### LINKED ISSUE #13476: suite-desktop start-on startup
consider implementing suite-desktop auto start. something like electron app.openAtLogin or similar. to be discussed. 

it is useful when we want to use suite-desktop as the source of bridge for 3rd parties.

### PR DESCRIPTION
**Pull request by martykan**
## Description

Add an option to settings to start Trezor Suite automatically as a background daemon to provide bridge.
Tested on Mac, Windows, Linux

## Considerations

- Tray icon
- How this affects update process

## Related Issue

Resolve #13476

### CODE REVIEW

File: packages/suite/src/views/settings/SettingsGeneral/AutoStart.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { ActionColumn, TextColumn, Translation } from 'src/components/suite';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { setAutoStart } from 'src/actions/suite/suiteActions';
+
+const PositionedSwitch = styled.div`
+    align-self: center;
+`;
+
+export const AutoStart = () => {
```
Comment (mroz22) [inline packages/suite/src/views/settings/SettingsGeneral/AutoStart.tsx]: maybe lets add one commit that moves it to debug section and that can be easily reverted later. This way we can postpone asking for permission to put it here 😄

**Comment by mroz22** [comment]
@marekrjpolak please check on it - whether it is in line with your desktop modules architecture 🙏

**Review by marekrjpolak — APPROVED** [review]
> please check on it - whether it is in line with your desktop modules architecture

I haven't tried or checked anything else, but the desktop module stuff seems to be implemented just fine. :ok_hand:

**Review by mroz22 — APPROVED** [review]
good, I briefly tested it on mac. and since this is opt-in in debug settings, I am pretty confident merging it.

**Pull request merged**